contracts/Diamond/libraries/LibHelper.sol

```diff
@@ -0,0 +1,97 @@
+pragma solidity 0.8.13;
+import "../../Tokens/VTokens/VToken.sol";
+import "./appStorage.sol";
+import "./LibAccessCheck.sol";
+import "../../Utils/ErrorReporter.sol";
+import "./LibExponentialNoError.sol";
+
+import "../../Utils/ExponentialNoError.sol";
+
+library LibHelper {
+    /// @notice The initial Venus index for a market
+    uint224 public constant venusInitialIndex = 1e36;
+    // closeFactorMantissa must be strictly greater than this value
+    uint internal constant closeFactorMinMantissa = 0.05e18; // 0.05
+    // closeFactorMantissa must not exceed this value
+    uint internal constant closeFactorMaxMantissa = 0.9e18; // 0.9
+    // No collateralFactorMantissa may exceed this value
+    uint internal constant collateralFactorMaxMantissa = 0.9e18; // 0.9
+
+    /**
+     * @notice Determine what the account liquidity would be if the given amounts were redeemed/borrowed
+     * @param vTokenModify The market to hypothetically redeem/borrow in
+     * @param account The account to determine liquidity for
+     * @param redeemTokens The number of tokens to hypothetically redeem
+     * @param borrowAmount The amount of underlying to hypothetically borrow
+     * @dev Note that we calculate the exchangeRateStored for each collateral vToken using stored data,
+     *  without calculating accumulated interest.
+     * @return (possible error code,
+                hypothetical account liquidity in excess of collateral requirements,
+     *          hypothetical account shortfall below collateral requirements)
+     */
+    function getHypotheticalAccountLiquidityInternal(
+        address account,
+        VToken vTokenModify,
+        uint redeemTokens,
+        uint borrowAmount
+    ) internal view returns (ComptrollerErrorReporter.Error, uint, uint) {
+        AppStorage storage s = LibAppStorage.diamondStorage();
+        (uint err, uint liquidity, uint shortfall) = s.comptrollerLens.getHypotheticalAccountLiquidity(
+            address(this),
+            account,
+            vTokenModify,
+            redeemTokens,
+            borrowAmount
+        );
```

Comment:
I think we can get rid of ComptrollerLens now since we have the Diamond architecture. The only purpose of this contract was to move a part of the functionality out of the Comptroller.

What do you think about moving the functions from lens back here to libHelper or to some of the facets?

Reply:
Sounds good, even we can make a new separate Facet for lens' methods. Wdyt?

Reply:
RN we have just moved the comptroller logic into facets,